extensions/test/algorithms/parse.cpp

```diff
@@ -104,6 +106,26 @@ void test_all_complex_dms_strategy()
 
 	test_parse_with_point_and_parse_strategy<P, parser_t>(std::string("5.15R E"), std::string("5.15R N"),
 		expected_dms_result9, expected_dms_result10, parser_t());
+
+	// test with random characters in strategy(for generality check)
+	typedef bg::projections::detail::dms_parser<coord_t, true, 'A', 'B', 'C', 'D', 'm', 's', 'o', 'p'> parser_t1;
```

Comment:
Since we moved to C++14, we should prefer
```
using parser_t1 = bg::projections::detail::dms_parser<coord_t, true, 'A', 'B', 'C', 'D', 'm', 's', 'o', 'p'>;
```

which places the alias name to the left, it's easier to read.